ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -241,15 +240,7 @@ public void requestBannerAd(Context context,
             return;
         }
 
-        //workaround for missing onPause/onResume/onDestroy
-        adLayout = new RelativeLayout(context) {
-            @Override
-            protected void onWindowVisibilityChanged(int visibility) {

```

Comment:
From AdMob SDK 18.2.0 all devices can receive onPause() and onResume() callback. So onWindowVisibilityChanged() doesn't needed.